src/api/constants.ts

```diff
@@ -0,0 +1,15 @@
+import { z } from "zod";
+import { schemaUserAPI } from "./User";
+
+export const API_KEY = process.env.NEXT_PUBLIC_API_KEY;
+export const API_URL = process.env.NEXT_PUBLIC_API_URL;
+
+export const config = {
+  headers: {
+    Authorization: `Bearer ${API_KEY}`,
```

Comment:
Here should a jwt token and if you store in the env then it should not be stored there